src/modules/dashboard/components/ExpenditurePage/Streaming/FundingSource/FundingSource.tsx

```diff
@@ -3,58 +3,27 @@ import { defineMessages } from 'react-intl';
 import classNames from 'classnames';
 import { parseInt } from 'lodash';
 import { ROOT_DOMAIN_ID } from '@colony/colony-js';
+import { useField } from 'formik';
 
 import { Colony } from '~data/index';
-import {
-  FormSection,
-  Input,
-  InputLabel,
-  SelectHorizontal,
-  SelectOption,
-  TokenSymbolSelector,
-} from '~core/Fields';
+import { FormSection, InputLabel, SelectOption } from '~core/Fields';
 import DomainDropdown from '~core/DomainDropdown';
 import { COLONY_TOTAL_BALANCE_DOMAIN_ID } from '~constants';
 import ColorTag, { Color } from '~core/ColorTag';
-import TokenIcon from '~dashboard/HookedTokenIcon';
+
+import { ExpenditureEndDateTypes } from '~pages/ExpenditurePage/types';
 
 import { FundingSource as FundingSourceType } from '../types';
 
+import Rate from '../Rate';
 import styles from './FundingSource.css';
+import Limit from '../Limit';
```

Comment:
Can you please reorder these inputs according to our system: 
move the `../` up & together without the empty line